apps/wallet-mobile/src/features/Menu/Menu.tsx

```diff
@@ -39,9 +39,10 @@ export const MenuNavigator = () => {
 
 export const Menu = () => {
   const strings = useStrings()
-  const styles = useStyles()
+  const {styles, color} = useStyles()
   const navigateTo = useNavigateTo()
   const wallet = useSelectedWallet()
+  const {poolTransition} = usePoolTransition()
```

Comment:
that's why the tx creation should be on fly, if the pool is retiring, at this point the tx was created, and eventually it can even fail.

Reply:
isPoolRetiring

Reply:
Sure, we can add a boolean to indicate whether the poolTransition is not null. But in this case, doesn't affect the tx creation.